### DISCUSSION TITLE
Accept in copilot

### DISCUSSION THREAD

**Discussion by ebrahimmelngary** [community/community#8315] [Copilot Conversations] [answered]
Hi,
am signup for a long time in copilot but not added ,
please add me ebrahim.elngary@gmail.com

**Comment by jakobroenn02** [comment]
no.

**Comment by davecheney** [comment, marked as answer]
@ebrahimmelngary We are inviting users as we have capacity. Thank you for your patience and understanding.

**Reply by UtkuX** [reply]
> @ebrahimmelngary We are inviting users as we have capacity. Thank you for your patience and understanding.

@davecheney Hi! I am impatiently waiting for the invitation. What is the latest situation in capacity and invitations? Do you have exact data? Will i wait any longer? waiting is very tiring and inefficient, please help! Thanks.

**Reply by ebrahimmelngary** [reply]
hope to be invite today before tomorrow

**Comment by ebrahimmelngary** [comment]
Thank you for your response hope to invite soon
